### DISCUSSION TITLE
Data Flows Export\Import

### DISCUSSION THREAD

**Discussion by meditadvisors** [directus/directus#13746] [Feature Requests]
The new data flows functionality is great and has a lot of potential!
 
It would be good to be able to export/import flows via the app and/or snapshots/migrations

My current workaround is to use SQL inserts with the 2 relevant tables (directus_flows and directus_operations). 
This works with variable results between (docker) self-hosted deployments...

Long term, this functionality may help to grow an app flow marketplace!

Thanks!

**Comment by VaZark** [comment]
That sounds interesting. That sounds exactly like Zapier or at least an initial prototype of it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by moseskamau338** [reply]
I think people need this now

**Reply by rijkvanzanten** [reply]
This is being included in the discussions for https://github.com/directus/directus/discussions/13041 🙂
